ZenPacks/zenoss/Docker/parsers/cgroupfs.py

```diff
@@ -178,9 +178,15 @@ class cgroupfs(CommandParser):
 
     createDefaultEventUsingExitCode = False
 
+    def dataForParser(self, context, dp):
+        return {
+            'cgroup_path': context.cgroup_path,
```

Comment:
This may not be safe because the new parser code could be loaded by zencommand after upgrading the ZenPack on a cached config where context doesn't have a cgroup_path attribute, so you'd get an AttributeError here. The safer way to do this for newly-added properties is like this.

```
return {
    "cgroup_path": getattr(context, "cgroup_path", "/sys/fs/cgroup"),
}
```